docs/node/CATEGORIES/16-loan.md

```diff
@@ -728,4 +729,55 @@ interface ListAuctionHistoryDetail {
   auctionBid: string
   auctionWon: string[]
 }
+```
+
+## listVaultHistory
+
+Returns the history of the specified vault.
+
+```ts title="client.loan.listVaultHistory()"
+interface loan {
+  listVaultHistory (vaultId: string, pagination?: ListVaultHistoryPagination): Promise<ListVaultHistory[]>
+}
+
+interface ListVaultHistoryPagination {
+  maxBlockHeight?: number
+  depth?: number
+  token?: string
+  txType?: string
+  limit?: number
+}
+
+interface ListVaultHistory {
+  blockHeight: number
+  blockHash: string
+  blockTime: number
+  type: string
+  txid: string
+  address?: string,
+  amounts?: string[], // amount@symbol
+  loanScheme?: CreateLoanScheme
+  vaultSnapshot?: VaultHistorySnapshot
+}
+
+interface CreateLoanScheme {
+  minColRatio: number
+  interestRate: BigNumber
```

Comment:
C++ side return this as rate. Need to tell them to use interestRate like how other RPC does